.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+      - name: Prepare variables for PR
+        # language=sh
+        run: |
+          release_line=$(grep 'release = \"' core/driver/sources/src/main/resources/reference.conf)
+          intellij_version=$(awk -F'\"|\"' '{print $2}' <<< "$release_line")
+          echo "BRANCH_NAME=update_intellij_version_to_$intellij_version" >> $GITHUB_ENV
+          echo "PR_TITLE=Update IntelliJ version to $intellij_version" >> $GITHUB_ENV
+          scala_plugin_file_url="https://github.com/VirtusLab/ide-probe/blob/master/extensions/scala/tests/src/test/resources/scala.conf"
+          scala_plugin_versions_url="https://plugins.jetbrains.com/plugin/1347-scala/versions"
+          echo "PR_BODY=Updates the default IntelliJ Version to the newest release. \
+            Please, check currently used version of the scala plugin: $scala_plugin_file_url \
+            and verify if it is compatible with the new intelliJ IDEA version ($intellij_version) here: $scala_plugin_versions_url. \
+            If the version of the scala plugin used in $scala_plugin_file_url is NOT compatible - please update it in a separate commit." >> $GITHUB_ENV
+      - name: Create Pull Request
```

Comment:
> What if a branch of this name (and/or a PR for such branch) already exists?

Is this handled automagically by this action?

Reply:
I'm not sure, but it looks like it might have such powers: https://github.com/peter-evans/create-pull-request#-create-pull-request
<img width="872" alt="image" src="https://user-images.githubusercontent.com/49597713/190678812-97c51306-9725-4781-b3e3-2bdb9fb79d96.png">

Anyway - let me test this scenario on my testing repo

Reply:
ok, looks like there is a good answer in the action's docs:
```
How the action behaves:

If there are changes (i.e. a diff exists with the checked-out base branch), the changes will be pushed to a new branch and a pull request created.
If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.
If a pull request already exists it will be updated if necessary. Local changes in the Actions workspace, or changes on the base branch, can cause an update. If no update is required the action exits silently.
If a pull request exists and new changes on the base branch make the pull request unnecessary (i.e. there is no longer a diff between the pull request branch and the base), the pull request is automatically closed. Additionally, if delete-branch is set to true the branch will be deleted.
```

from https://github.com/peter-evans/create-pull-request#action-behaviour 

TL;DR - **yes** - such scenarios are handled automagically by this action